engine/battle/core.asm

```diff
@@ -5173,6 +5173,7 @@ MoveSelectionScreen:
 	jmp MoveSelectionScreen
 
 .struggle
+	call ClearSprites
```

Comment:
This is typically fine to use; but keep in mind it clears **_ALL_** sprites. I think when you are calling this, there are no sprites on screen other than the weather image. Unless someone thinks of an edge case; should be fine.